disdrodb/L0/readers/ARM/ARM_lpm.py

```diff
@@ -8,59 +8,27 @@
 
 import os
 import time
-import click
 
 # Directory
-from disdrodb.io import check_directories
-from disdrodb.io import get_campaign_name
-from disdrodb.io import create_directory_structure
+from disdrodb.L0.io import check_directories
+from disdrodb.L0.io import get_campaign_name
+from disdrodb.L0.io import create_directory_structure
 
 # Logger
-from disdrodb.logger import create_L0_logger
-from disdrodb.logger import close_logger
+from disdrodb.utils.logger import create_L0_logger
+from disdrodb.utils.logger import close_logger
 
 # Metadata
-from disdrodb.metadata import read_metadata
-from disdrodb.check_standards import check_sensor_name
+from disdrodb.L0.metadata import read_metadata
+from disdrodb.L0.check_standards import check_sensor_name
 
 # L0 processing
-from disdrodb.L0_proc import get_file_list
-from disdrodb.io import get_L1_netcdf_fpath
-from disdrodb.L1_proc import write_L1_to_netcdf
-
-# -------------------------------------------------------------------------.
-# CLIck Command Line Interface decorator
-@click.command()  # options_metavar='<options>'
-@click.argument("raw_dir", type=click.Path(exists=True), metavar="<raw_dir>")
-@click.argument("processed_dir", metavar="<processed_dir>")
-@click.option(
-    "-f",
-    "--force",
-    type=bool,
-    show_default=True,
-    default=False,
-    help="Force overwriting",
-)
-@click.option(
-    "-v", "--verbose", type=bool, show_default=True, default=False, help="Verbose"
-)
-@click.option(
-    "-d",
-    "--debugging_mode",
-    type=bool,
-    show_default=True,
-    default=False,
-    help="Switch to debugging mode",
-)
-@click.option(
-    "-l",
-    "--lazy",
-    type=bool,
-    show_default=True,
-    default=True,
-    help="Use dask if lazy=True",
-)
-def main(
+from disdrodb.L0.L0A_processing import get_file_list
+from disdrodb.L0.io import get_L1_netcdf_fpath
```

Comment:
Update readers to use `get_L0B_fpath` instead of  `get_L1_netcdf_fpath`

Reply:
ok